packages/components/src/components/teaser-media/teaser-media.stories.ts

```diff
@@ -18,242 +29,173 @@ export default {
     {
       type: 'slot',
       name: 'media',
-      value: `<div slot="media" class="slot slot--border slot--text h-[384px]">Media slot</div>`
+      value: `<img slot="media" src="./placeholders/images/generic.jpg" class="aspect-video object-cover" alt="Test"/>`
     },
     {
       type: 'slot',
       name: 'meta',
       value: `<div slot="meta" class="slot slot--border slot--text h-12">Meta slot</div>`
+    },
+    {
+      type: 'slot',
+      name: 'headline',
+      value: `<div slot="headline" class="h-12">Headline Media Teaser</div>`
     }
   ]),
   argTypes,
   parameters
 };
 
-/**
- * This shows sd-teaser-media in its default state.
- */
-
 export const Default = {
   render: (args: any) => {
     return generateTemplate({ args });
   }
 };
 
 /**
- * Teaser-Media in all possible `variant`.
+ * Use the `variant` attribute to set the color variant:
+ * - `white (default)`
+ * - `gradient-dark`
+ * - `gradient-white`
+ * - `primary`
+ * - `primary-100`
+ * - `neutral-100`
```

Comment:
Please change the order to:
<img width="218" alt="image" src="https://github.com/user-attachments/assets/00a7a695-7207-4f88-bf3a-df46e37f94d5">